scripts/download-vscode.js

```diff
@@ -17,9 +17,11 @@ const extensionDirectories = [
   'salesforcedx-vscode-visualforce'
 ];
 
+const vscodeVersion = process.env.CODE_VERSION || 'stable';
```

Comment:
The one thing kind of annoying about using an environment variable for this is we can't see the version in the output logs since CircleCI masks them...

Reply:
You can, it gets printed as part of the output of `vscode-test` when it downloads vscode.

Reply:
```
Downloading VS Code ****** from https://update.code.visualstudio.com/******/win32-archive/stable
Downloaded VS Code ****** into .vscode-test/vscode-******
Executable Path: C:\Users\circleci\project\.vscode-test\vscode-******\Code.exe
Directory Name: vscode-******
Base Path: C:\Users\circleci\project
Full Path: C:\Users\circleci\project\.vscode-test\vscode-******\
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-apex\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-apex\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-apex-debugger\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-apex-debugger\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-apex-replay-debugger\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-apex-replay-debugger\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-core\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-core\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-lightning\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-lightning\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-lwc\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-lwc\.vscode-test\vscode-******
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-visualforce\.vscode-test\vscode-******
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-visualforce\.vscode-test\vscode-******
```

i cant find it when i search it either :(

Reply:
I see what you mean, I think CircleCI is smart enough to recognize the value was set from an env variable and masks it. Here's a sample from a branch without these changes:

```
> @ test:integration-tests-without-system-tests C:\Users\circleci\project
> node scripts/download-vscode && lerna exec --ignore system-tests --concurrency 1 --stream --bail=false -- npm run test:integration --if-present && npm run test:vscode-integration --if-present

Downloading VS Code 1.53.2 from https://update.code.visualstudio.com/1.53.2/win32-archive/stable
Downloaded VS Code 1.53.2 into .vscode-test/vscode-1.53.2
Executable Path: C:\Users\circleci\project\.vscode-test\vscode-1.53.2\Code.exe
Directory Name: vscode-1.53.2
Base Path: C:\Users\circleci\project
Full Path: C:\Users\circleci\project\.vscode-test\vscode-1.53.2\
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-apex\.vscode-test\vscode-1.53.2
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-apex\.vscode-test\vscode-1.53.2
Creating Directories:  C:\Users\circleci\project\packages\salesforcedx-vscode-apex-debugger\.vscode-test\vscode-1.53.2
Copying to: C:\Users\circleci\project\packages\salesforcedx-vscode-apex-debugger\.vscode-test\vscode-1.53.2
```